esm-collection/README.md

```diff
@@ -0,0 +1,61 @@
+# ESM Collection Specification
+
+- **Title: ESM Collection**
+- **Identifier: esm**
+- **Field Name Prefix: esm**
+- **Scope: Collection**
+- **Extension [Maturity Classification](https://github.com/radiantearth/stac-spec/blob/master/extensions/README.md#extension-maturity): Proposal**
+
+This document explains the fields of the STAC Earth System Model (ESM) Extension to a STAC Collection.
+ESM data is considered to be data produced by simulations models of the earth for a data specific model domain and temporal extent.
+
+- [Example](./examples/sample-pangeo-cmip6-collection.json)
+- [JSON Schema](./json-schema/schema.json)
+
+## Collection Fields
+
+This extension introduces a three new fields at the top level of the collection: `esm:catalog`, `esm:attributes` and `esm:aggregation_control`.
```

Comment:
@m-mohr, I could use your input here :) 

I am trying to figure out whether the following fields belong in this section of the README?

- `type`
- `roles`
- `esm:data_field`
- `esm:format`
- `esm:attributed_field`

Reply:
Hmm, I guess type and roles doesn't belong here... they have no prefix and are likely for assets.

The others (esm:...) likely belong here, but it was a while ago we did this and I can't fully remember what all these fielda are about.

Reply:
Thank you for your feedback! 

> I need to think into this again, but it's already 10PM here so will postpone that ;-)

I understand. Are you around tomorrow or next week? I'd like to ping you for additional help on this :)

Reply:
I'm off work until Tuesday, but I'm looking into mails/Gitter etc every once in a while to help out during the data sprint.
I guess best would be to have a short call tomorrow (can use the data sprint main room) or so to discuss a bit and then you could likely finish it off.

Reply:
Awesome! Yes, a short call would be helpful. Let me know what time works best for you tomorrow and we shall schedule something. My schedule is flexible tomorrow.

Reply:
What timezone are you in? Would 14:00 UTC work for you?

Reply:
I am in Mountain Daylight Time zone. 14:00 UTC (8:00 AM local time) works for me

![Screen Shot 2020-09-10 at 2 58 35 PM](https://user-images.githubusercontent.com/13301940/92805325-3f2fa900-f376-11ea-8888-5aa028c703c9.png)

Reply:
We can also do 1 or 2 hours later if that's better for you.

Reply:
`14:00 UTC` is a good one (before getting distracted by other tasks later in the day :) )